make-cd/Makefile

```diff
@@ -18,7 +21,7 @@ burn-audio: audio.raw
 	touch burn-audio
 
 1st_read.bin: $(1ST_READ)
-	$(SCRAMBLE) $(1ST_READ) 1st_read.bin
+	cp $(1ST_READ) 1st_read.bin
```

Comment:
Why did the call to scramble get taken out? This will produce a non-bootable disc as it is.

Reply:
Brought back scramble